vhdbuilder/packer/install-dependencies.sh

```diff
@@ -619,3 +619,40 @@ rm -f ./azcopy # cleanup immediately after usage will return in two downloads
 echo "install-dependencies step completed successfully"
 capture_benchmark "${SCRIPT_NAME}_overall" true
 process_benchmarks
+
+
+download_amdgpu_drivers() {
```

Comment:
do we have a device-plugin to install as well to ensure the GPU count shows up on the NOde labels ?

Reply:
It exists, but probably not in MCR. 

AFAIK we doesn't do it yet for Nvidia. Should be consistent for both of them? 

We probably can addess it separatelly.